package/MDAnalysis/coordinates/base.py

```diff
@@ -152,22 +152,24 @@ class Timestep(object):
     def __init__(self, n_atoms, **kwargs):
         """Create a Timestep, representing a frame of a trajectory
 
-        :Arguments:
-          *n_atoms*
-            The total number of atoms this Timestep describes
-
-        :Keywords:
-          *positions*
-            Whether this Timestep has position information [``True``]
-          *velocities*
-            Whether this Timestep has velocity information [``False``]
-          *forces*
-            Whether this Timestep has force information [``False``]
-          *dt*
-            The time difference between frames (ps).  If :attr:`time`
-            is set, then `dt` will be ignored.
-          *time_offset*
-            The starting time from which to calculate time (ps)
+        Arguments
+        ---------
+        *n_atoms*
+          The total number of atoms this Timestep describes
+
+        Keyword
+        -------
+        *positions*
+          Whether this Timestep has position information [``True``]
+        *velocities*
+          Whether this Timestep has velocity information [``False``]
+        *forces*
+          Whether this Timestep has force information [``False``]
+        *dt*
+          The time difference between frames (ps).  If :attr:`time`
+          is set, then `dt` will be ignored.
+        *time_offset*
+          The starting time from which to calculate time (ps)
```

Comment:
[Numpy style doc-strings](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) work a little bit different. All function parameters are under the `Parameters` heading. For keyword arguments you are supposed to add `optional` behind the type. So the correct numpy formating for the parameter list is.
 
```
Parameters
----------
n_atoms: int
  The total number of atoms this Timestep describes
positions: bool, optional
  Whether this Timestep has position information [``True``]
velocities: bool, optional
  Whether this Timestep has velocity information [``False``]
forces: bool, optional
  Whether this Timestep has force information [``False``]
dt: float, optional
  The time difference between frames (ps).  If :attr:`time`
  is set, then `dt` will be ignored.
time_offset: float, optional
  The starting time from which to calculate time (ps)
```